--- MonoBook.php.orig	2007-05-11 11:12:56.000000000 +0200
+++ MonoBook.php.patched	2007-05-11 11:24:51.000000000 +0200
@@ -155,7 +155,11 @@
 		<h5><?php $out = wfMsg( $bar ); if (wfEmptyMsg($bar, $out)) echo $bar; else echo $out; ?></h5>
 		<div class='pBody'>
 			<ul>
-<?php 			foreach($cont as $key => $val) { ?>
+<?php			global $wgVoWiCustomNavBlocksEnable;
+			if ($wgVoWiCustomNavBlocksEnable == "true")
+				include ('extensions/customnavblocks.php');
+			else
+			foreach($cont as $key => $val) { ?>
 				<li id="<?php echo Sanitizer::escapeId($val['id']) ?>"<?php
 					if ( $val['active'] ) { ?> class="active" <?php }
 				?>><a href="<?php echo htmlspecialchars($val['href']) ?>"><?php echo htmlspecialchars($val['text']) ?></a></li>
